apps/website/src/pages/docs/tile.mdx

```diff
@@ -14,14 +14,130 @@ import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-tile' />
-
 <LiveExample src='Tile.main.tsx'>
   <AllExamples.TileMainExample client:load />
 </LiveExample>
 
 Tiles are containers that display content and actions pertaining to a single item or project. They should be easy to scan for relevant and actionable information. All text should be concise and placed in a way that clearly indicates informational hierarchy.
 
-## Props
+### Composition API
+
+Users can build Tiles with subcomponents that are fully customizable.
+
+We provide:
+
+- `Tile.Wrapper`
+- `Tile.ThumbnailArea`
+- `Tile.ThumbnailPicture`
+- `Tile.QuickAction`
+- `Tile.TypeIndicator`
+- `Tile.BadgeContainer`
+- `Tile.IconButton`
+- `Tile.Name`
+- `Tile.NameIcon`
+- `Tile.NameLabel`
+- `Tile.Action`
+- `Tile.ContentArea`
+- `Tile.Description`
+- `Tile.Metadata`
+- `Tile.MoreOptions`
+- `Tile.Buttons`
+
+<LiveExample src='Tile.composition.tsx'>
+  <AllExamples.TileCompositionExample client:load />
+</LiveExample>
+
+### Props API
+
+Users can also take advantage of properties provided for `Tile`. A list of available properties can be found [at the end](#props) of this documentation.
```

Comment:
if it helps, you could look at expandable block documentation for ideas on how to separate composition and legacy apis. https://github.com/iTwin/iTwinUI/pull/1502/files#diff-19c324894ed0750aa99b244219e53d7e123753f73b594a0211986918f7bf4e03

Reply:
Despite the wording, what's wrong with my choice of documenting both apis?

Reply:
The example is fine, but the wording just feels unhelpful in general.

"You can take advantage of props" is not some new revelation; users can always take advantage of props. It might help to describe these props or mention which ones.

But the key difference is between *directly* using `Tile` vs *only* using subcomponents. See "usage" section here: https://dev-itwinui.bentley.com/docs/expandableblock#usage

Reply:
Updated, is it better now?

Reply:
2/3 find this approach "good enough". 
Leaving as is for now.